src/picongpu/include/simulation_defines/param/fieldSolver.param

```diff
@@ -26,6 +26,22 @@
  *
  * You can set/modify Maxwell solver specific options in the
  * section of each "FieldSolver".
+ *
+ *
+ * \typedef CurrentInterpolation is used to set a method performing the
+ * interpolate/assign opteraion from the generated currents of particle
+ * species to the electro-magnetic fields.
+ *
+ * Allowed values are:
+ *   - None<simDim>:
+ *     - default for staggered grids/Yee-scheme
+ *     - updates E
+ *   - Binomial<simDim>: 2nd order Binomial filter
+ *     - smooth the current before assignment in staggered grid
```

Comment:
everywhere else you use the verbs in 3rd person singular - please change from `smooth` to `smooths`